### PR TITLE
Add full screen tests for test_app.py

### PR DESCRIPTION
**Pull request by tczorro**
Improved the coverage for test_app.py in toga.
Test contents:
    1. Test set_full_screen and exit_full_screen
    2. Test set_full_screen for both triggering and exiting

## PR Checklist:

- [x] All new features have been tested
- [x] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

**Comment by danyeaw** [comment]
@tczorro Can you please update this from pytest to unittest format?

File: src/core/tests/test_app.py [hunk 1]
```diff
@@ -65,6 +65,18 @@ def test_app_exit(self):
 
         self.assertActionPerformed(self.app, 'exit')
 
+    def test_full_screen(self):
+        # set full screen and exit full screen
+        self.app.set_full_screen(self.app.main_window)
+        self.assertTrue(self.app.is_full_screen)
+        self.app.exit_full_screen()
+        self.assertFalse( self.app.is_full_screen)
```
Comment (brutusthebee) [inline src/core/tests/test_app.py]: At column 26: (E201) whitespace after '('

**Comment by tczorro** [comment]
> @tczorro Can you please update this from pytest to unittest format?

Updated with unittest builtin assert functions.

File: src/core/tests/test_app.py (hunk 1 above)
Comment (tczorro) [reply src/core/tests/test_app.py]: got sloppy, my bad. Fixed

**Pull request merged**